site/src/content/docs/deploy/freestyle.mdx

```diff
@@ -0,0 +1,150 @@
+# FreeStyle
+
+Deploy Rivet Actors to [FreeStyle.sh](https://docs.freestyle.sh/), a cloud platform for running AI-generated code with built-in security and scalability.
+
+## Setup
+
+<Steps>
+<Step title="Install packages">
+
+Install RivetKit and create your registry:
+
+```bash
+npm install rivetkit
+```
+
+</Step>
+
+<Step title="Create namespace">
```

Comment:
guides are not rivet cloud-specific. these guides should not assume you need a new ns.

self-hotsed = use default namespace
cloud = we provide default prod ns